From a6b3cfb6b1bb5e585e059723570591ca18446930 Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Wed, 4 Jun 2014 00:51:15 +1000
Subject: [PATCH 06/16] Fix search of image artwork in folder

The first character of the path was dropped, causing the search for images to always fail
---
 mythtv/programs/mythfrontend/videodlg.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/programs/mythfrontend/videodlg.cpp b/mythtv/programs/mythfrontend/videodlg.cpp
index 861c396..ee344a0 100644
--- a/mythtv/programs/mythfrontend/videodlg.cpp
+++ b/mythtv/programs/mythfrontend/videodlg.cpp
@@ -1650,7 +1650,7 @@ QString VideoDialog::GetCoverImage(MythGenericTree *node)
                         QUrl sgurl = *iter;
                         QString path = sgurl.path();
 
-                        QString subdir = folder_path.right(folder_path.length() - (prefix.length() + 2));
+                        QString subdir = folder_path.right(folder_path.length() - (prefix.length() + 1));
 
                         path = path + "/" + subdir;
 
-- 
1.7.10.2

